LayoutTests/css-dark-mode/link-colors-expected.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+
+<style>
+
+.light {
+    color: #0000EE;
+}
+
+.dark {
+    color: #9E9EFF;
+}
+
+</style>
+
+<a class="light" href="https://www.webkit.org">Light</a>
+<a class="dark" href="https://www.webkit.org">Dark</a>
```

Comment:
You can also add tests for :visited by doing `<a href="">`

Reply:
Updated – thanks!